container/crio/handler.go

```diff
@@ -262,16 +262,21 @@ func (self *crioContainerHandler) getFsStats(stats *info.ContainerStats) error {
 		limit  uint64
 		fsType string
 	)
+	deviceFound := false
 
 	// crio does not impose any filesystem limits for containers. So use capacity as limit.
 	for _, fs := range mi.Filesystems {
 		if fs.Device == device {
 			limit = fs.Capacity
 			fsType = fs.Type
+			deviceFound = true
 			break
 		}
 	}
 
+	if !deviceFound {
```

Comment:
I would prefer checking if `fsType == ""`, rather than using a bool in the loop

Reply:
@dashpole 
Please take another look.